src/Compilers/CSharp/Portable/Syntax/SyntaxKind.cs

```diff
@@ -4,6 +4,8 @@
 
 #nullable enable
 
+using Microsoft.CodeAnalysis.CSharp.Syntax;
```

Comment:
@sharwell, CI is failing here with error `CS0234 The type or namespace name 'Syntax' does not exist in the namespace 'Microsoft.CodeAnalysis.CSharp'`

This doesn't happen with me in Visual Studio.

Reply:
I believe this is occurring because `SyntaxKinds` is linked into the code generation projects. You may need to wrap this in `#if` so it's only used for the Microsoft.CodeAnalysis.CSharp project.

Reply:
@sharwell, *Microsoft.CodeAnalysis.CSharp.csproj* doesn't currently contain any `DefineConstants`. Do I need to add a new one and use if for the `#if` directive?

Reply:
The other option is to use a fully-qualified `cref` attribute:

```
<see cref="T:Microsoft.CodeAnalysis.CSharp.Syntax.OmittedTypeArgumentSyntax"/>
```